src/main/java/cloud/fogbow/ras/core/RasClassFactory.java

```diff
@@ -27,6 +30,11 @@ public Object createPluginInstance(String pluginClassName, String ... params)
             }
 
             pluginInstance = constructor.newInstance(params);
+            
+        	if (pluginClassName.equals("cloud.fogbow.common.plugins.authorization.ComposedAuthorizationPlugin")) {
+        		ComposedAuthorizationPlugin<RasOperation> composedPlugin = (ComposedAuthorizationPlugin<RasOperation>) pluginInstance;
+        		composedPlugin.startPlugin(this);
+        	}
```

Comment:
Smells bad. At a minimum I would move this code to the getAuthorizationPlugin() method in AuthorizationPluginInstantiator.

Reply:
Moved to AuthorizationPluginInstantiator.